Dashboard/Views/FooterView.axaml

```diff
@@ -0,0 +1,69 @@
+﻿<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="100"
+             xmlns:vm="clr-namespace:Dashboard.ViewModels"
+             x:Class="Dashboard.Views.FooterView"
+             x:DataType="vm:FooterViewModel"
+             Background="#ff2b2b2b"
+             Foreground="White">
+    <UserControl.Styles>
+        <!-- Style to remove Avalonia's default hover and selected effects for Textboxes -->
+        <Style Selector="TextBox">
+            <Style Selector="^:focus">
+                <Setter Property="BorderBrush" Value="Red"/>
+                        <Style Selector="^ /template/ Border#PART_BorderElement">
+                        <Setter Property="Background" Value="White"/>
+                        <Setter Property="BorderBrush" Value="White"/>
+                        <Setter Property="BorderThickness" Value="1" />
+                    </Style>
+                </Style>
+            <Style Selector="^:pointerover">
+                <Setter Property="BorderBrush" Value="Red"/>
+                <Style Selector="^ /template/ Border#PART_BorderElement">
+                    <Setter Property="Background" Value="White"/>
+                    <Setter Property="BorderBrush" Value="White"/>
+                    <Setter Property="BorderThickness" Value="1" />
+                </Style>
+            </Style>
+        </Style>
+    </UserControl.Styles>
+    <Design.DataContext>
+        <!-- This only sets the DataContext for the previewer in an IDE,
+             to set the actual DataContext for runtime, set the DataContext property in code (look at App.axaml.cs) -->
+        <vm:FooterViewModel />
+    </Design.DataContext>
+    <Grid ColumnDefinitions="0.2*,0.2*, 0.6*" >
+        <Grid Grid.Column="0">
+            <StackPanel VerticalAlignment="Center">
```

Comment:
Can you play around with the padding / margins plz:
- On the left of the times can you match the same margin that is to the right of hte console box
- Between the radios can you add a bit of padding

Reply:
Can do.

Reply:
https://github.com/nickowen44/AV-Datalogger/pull/10/commits/af9d6f189027cdae3bff3d19679e23588c2190d4